src/components/WizardModal/WizardModal.test.jsx

```diff
@@ -156,32 +156,36 @@ describe('WizardModal', () => {
     expect(wrapper.find(Loading)).toHaveLength(1);
   });
   it('clicking on progressIndicator steps will render related content', () => {
-    render(<WizardModal {...commonWizardProps} steps={[
-      { label: 'step1', content: 'page 1' },
-      { label: 'step2', content: 'page 2', onValidate: () => false },
-      { label: 'step3', content: 'page 3' },
-    ]} />);
-
-      // analogue to ProgressIndicator test we check if clicking step will show related content
-      const [ beforeClick1 ] = screen.getByTitle('step1').children;
-    
-      screen.getByTestId('iot--progress-step-button-main-step2').click();
-    
-      // content should be page2
-      expect(screen.getByTitle('step1').children[0]).not.toContain(beforeClick1);
-      expect(screen.getByText('page 2')).toBeDefined();
-
-      // clicking on step3 should not progress the modal
-      const [ beforeClick2 ] = screen.getByTitle('step2').children;
-      screen.getByTestId('iot--progress-step-button-main-step3').click();
-      
-      expect(screen.getByTitle('step2').children[0]).toEqual(beforeClick2);
-      expect(screen.getByText('page 2')).toBeDefined();
-
-      // clicking on page 1 should go back to step 1
-      screen.getByTestId('iot--progress-step-button-main-step1').click();
-      expect(screen.getByTitle('step1').children[0]).toEqual(beforeClick1);
-      expect(screen.getByText('page 1')).toBeDefined();
+    render(
```

Comment:
I also got this but excluded it from my PR.  It looks like auto-formatting from Prettier and perhaps we should just commit this and get rid of it. It is a bit strange that is is formatted since we don't touch the file...

Reply:
Yes, please leave it in. This was introduced in https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1611

Status checks must've been bypassed locally by the author. I'll look into adding a prettier validation to our Travis builds to prevent in the future https://github.com/carbon-design-system/carbon-addons-iot-react/issues/1659